eng/New-WindowsInstaller.ps1

```
@@ -0,0 +1,191 @@
+        $(if (-not [string]::IsNullOrWhiteSpace($programFiles)) { Join-Path $programFiles "Inno Setup 6\ISCC.exe" })
+    ) | Where-Object { -not [string]::IsNullOrWhiteSpace($_) }
+
+    foreach ($candidate in $candidates) {
+        if (-not [string]::IsNullOrWhiteSpace($candidate) -and
+            (Test-Path -LiteralPath $candidate)) {
+            return $candidate
+        }
+    }
+
+    $command = Get-Command "ISCC.exe" -CommandType Application -ErrorAction SilentlyContinue
+    if ($command is not $null) {
```

PSScriptAnalyzer (1.25.0)

[error] Unexpected token 'is' in expression or statement.

(UnexpectedToken)